newsletter/admin.py

```diff
@@ -516,13 +620,14 @@ def get_urls(self):
         # only used in this part of the admin. For now, leave them here.
         if HAS_CBV_JSCAT:
             my_urls.append(url(r'^jsi18n/$',
-                JavaScriptCatalog.as_view(packages=('newsletter',)),
-                name='newsletter_js18n'))
+                               JavaScriptCatalog.as_view(
```

Comment:
Please rebase on current master (this was fixed slightly different in another PR)